### PR TITLE
Discovering and learning everything there is to know about R packages using r-universe

### PR DESCRIPTION
**Pull request by jeroen**
Based on the new gallery widget 😄 
Preview: https://deploy-preview-490--ropensci.netlify.app/blog/2023/02/24/runiverse-discovering/

### CODE REVIEW

**Review by yabellini — COMMENTED** [review]
Review until ## nivel 1 title.

File: content/blog/2023-02-24-runiverse-discovering-es/index.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+
+Estas páginas de paquetes se generan automáticamente y proporcionan información detallada sobre cada proyecto, así como toda la información y recursos para instalar y probar rápidamente el paquete localmente en R.
+
+### Información del paquete
+
+La sección inicial de la página de inicio muestra datos descriptivos y enlaces a recursos de este paquete, como los archivos fuente/binarios, el manual del paquete y el archivo NEWS. Más abajo mostramos las instrucciones de instalación para instalar el paquete en R. La instalación es rápida y sencilla porque proporcionamos binarios precompilados para Windows y MacOS, por lo que no se necesitan herramientas ni configuraciones especiales en la máquina del usuario.
+
+{{< gallery >}}
+{{< figureforgallery src="../runiverse-discovering/package2.png" alt="Screenshot of r-universe package homepage with annotations." >}}
+{{< figureforgallery src="../runiverse-discovering/package21.png" alt="Screenshot of r-universe package badges and statistics." >}}
```
Comment (yabellini) [inline content/blog/2023-02-24-runiverse-discovering-es/index.md:94]:
```suggestion
{{< figureforgallery src="../runiverse-discovering/package21.png" alt="Ejemplo de una página mostrando las insignias y estadísticas de un paquete en r-universe." >}}
```

File: content/blog/2023-02-24-runiverse-discovering-es/index.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+{{< figureforgallery src="../runiverse-discovering/package4.png" alt="Screenshot of r-universe package: how to cite the package." >}}
```
Comment (yabellini) [inline content/blog/2023-02-24-runiverse-discovering-es/index.md:95]:
```suggestion
{{< figureforgallery src="../runiverse-discovering/package4.png" alt="Ejemplo de como se presenta la información de como citar el paquete en r-universe." >}}
```

File: content/blog/2023-02-24-runiverse-discovering-es/index.md [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+{{< /gallery >}}
+
+Aquí también puedes encontrar más información derivada por el sistema de construcción, como sobre qué bibliotecas del sistema construye el paquete, temas y si el paquete está en CRAN. Las etiquetas de debajo muestran estadísticas sobre el paquete, como dependencias, dependencias inversas y estrellas de GitHub. Haciendo clic en la etiqueta "exportaciones" se muestran todas las funciones y conjuntos de datos exportados por el paquete, que enlazan con las secciones respectivas de la página del manual. Si se especifica, también mostramos cómo debe citarse el paquete en los artículos.
```
Comment (yabellini) [inline content/blog/2023-02-24-runiverse-discovering-es/index.md:98]:
```suggestion
Aquí también puedes encontrar más información derivada por el sistema de construcción, como sobre qué bibliotecas del sistema construye el paquete, temas y si el paquete está en CRAN. Las etiquetas debajo muestran estadísticas sobre el paquete, como dependencias, dependencias inversas y estrellas de GitHub. Haciendo clic en la etiqueta "exports" se muestran todas las funciones y conjuntos de datos exportados por el paquete, que enlazan con las secciones respectivas de la página del manual. Si se especifica, también mostramos cómo debe citarse el paquete en los artículos.
```

File: content/blog/2023-02-24-runiverse-discovering-es/index.md [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+
+### Toda la documentación
+
+Para que puedas empezar a utilizar los paquetes lo antes posible, r-universe ofrece toda la documentación disponible y la presenta de la forma más clara y concisa posible en la página de inicio.
+
+La mayoría de los paquetes de R tienen 3 formas de documentación: un archivo léeme, un manual de referencia (páginas de ayuda) y una serie de artículos (viñetas). Estos artículos suelen ser un buen punto de partida, y aparecen en la página principal del paquete, justo debajo de los descriptivos. Desde aquí puedes ver tanto el documento html/pdf renderizado, como el archivo fuente rmarkdown utilizado para (re)producir el artículo.
```
Comment (yabellini) [inline content/blog/2023-02-24-runiverse-discovering-es/index.md:104]:
```suggestion
La mayoría de los paquetes de R tienen 3 formas de documentación: un archivo léeme (README), un manual de referencia (páginas de ayuda) y una serie de artículos (viñetas). Estos artículos suelen ser un buen punto de partida, y aparecen en la página principal del paquete, justo debajo de los descriptivos. Desde aquí puedes ver tanto el documento html/pdf renderizado, como el archivo fuente rmarkdown utilizado para (re)producir el artículo.
```

File: content/blog/2023-02-24-runiverse-discovering-es/index.md [hunk 5]
```diff
@@ -0,0 +1,135 @@
...
+
+{{< gallery >}}
+{{< figureforgallery src="../runiverse-discovering/package3.png" alt="Screenshot of r-universe package, listing included articles." >}}
```
Comment (yabellini) [inline content/blog/2023-02-24-runiverse-discovering-es/index.md:107]:
```suggestion
{{< figureforgallery src="../runiverse-discovering/package3.png" alt="Ejemplo del universo de un paquete listando todos los artículos incluidos." >}}
```

File: content/blog/2023-02-24-runiverse-discovering-es/index.md [hunk 6]
```diff
@@ -0,0 +1,135 @@
...
+{{< figureforgallery src="../runiverse-discovering/package5.png" alt="Screenshot of r-universe package showing example readme content." >}}
```
Comment (yabellini) [inline content/blog/2023-02-24-runiverse-discovering-es/index.md:108]:
```suggestion
{{< figureforgallery src="../runiverse-discovering/package5.png" alt="Ejemplo del contenido de un archivo README en el univrso de un paquete." >}}
```

File: content/blog/2023-02-24-runiverse-discovering-es/index.md [hunk 7]
```diff
@@ -0,0 +1,135 @@
...
+{{< figureforgallery src="../runiverse-discovering/package6.png" alt="Screenshot of r-universe package showing reference manual index." >}}
```
Comment (yabellini) [inline content/blog/2023-02-24-runiverse-discovering-es/index.md:109]:
```suggestion
{{< figureforgallery src="../runiverse-discovering/package6.png" alt="Ejemplo de la referencia al índice de un manual de un paquete en r-universe." >}}
```

File: content/blog/2023-02-24-runiverse-discovering-es/index.md [hunk 8]
```diff
@@ -0,0 +1,135 @@
...
+{{< /gallery >}}
+
+Más abajo puedes ver el contenido del "Léame" del paquete. Hoy en día, la mayoría de los paquetes incluyen un archivo README.md que se puede convertir en HTML.
```
Comment (yabellini) [inline content/blog/2023-02-24-runiverse-discovering-es/index.md:112]:
```suggestion
Más abajo puedes ver el contenido del "README" del paquete. Hoy en día, la mayoría de los paquetes incluyen un archivo README.md que se puede convertir en HTML.
```

File: content/blog/2023-02-24-runiverse-discovering-es/index.md [hunk 9]
```diff
@@ -0,0 +1,135 @@
...
+
+Después del readme sigue una sección llamada "manual de referencia" que contiene una tabla con todas las páginas de ayuda. Éstas enlazan con los respectivos capítulos del manual de referencia html[^2], que proporciona la documentación más extensa de cada función del paquete, incluyendo descripciones, parámetros, ejemplos, etc.
```
Comment (yabellini) [inline content/blog/2023-02-24-runiverse-discovering-es/index.md:114]:
```suggestion
Después del README sigue una sección llamada "manual de referencia" que contiene una tabla con todas las páginas de ayuda. Éstas enlazan con los respectivos capítulos del manual de referencia html[^2], que proporciona la documentación más extensa de cada función del paquete, incluyendo descripciones, parámetros, ejemplos, etc.
```

File: content/blog/2023-02-24-runiverse-discovering-es/index.md [hunk 10]
```diff
@@ -0,0 +1,135 @@
...
+
+{{< figureforgallery src="../runiverse-discovering/refman.png" alt="Screenshot of a chapter in the HTML reference manual for a package, showing descriptions, parameters, and examples for a function." >}}
```
Comment (yabellini) [inline content/blog/2023-02-24-runiverse-discovering-es/index.md:116]:
```suggestion
{{< figureforgallery src="../runiverse-discovering/refman.png" alt="Ejemplo de un capítulo en en el manual de referencia en HTML de un paquete, mostrando descripciones, parámetros y ejemplos de una función." >}}
```

File: content/blog/2023-02-24-runiverse-discovering-es/index.md [hunk 11]
```diff
@@ -0,0 +1,135 @@
...
+
+### Actividad de uso y desarrollo
+
+Además de familiarizarte con lo que hace el paquete, puede que te interese saber quién está trabajando en él, quién más lo utiliza y si sigue desarrollándose activamente.
+
+La sección "desarrollo y colaboradores" muestra un gráfico de barras con el número de commits por semana durante el último año, y quiénes son los principales colaboradores. Por último, la sección "uso por otros paquetes" muestra otros paquetes de r-universo que dependen de este paquete, agrupados por propietario. Esto te da una idea de lo activo que es el proyecto y de quién está implicado.
```
Comment (yabellini) [inline content/blog/2023-02-24-runiverse-discovering-es/index.md:122]:
```suggestion
La sección ""development and contributors" (desarrollo y colaboraciones) muestra un gráfico de barras con el número de commits por semana durante el último año, y quiénes son los y las principales colaboradores y colaboradoras. Por último, la sección "usage by other packages" (uso por otros paquetes) muestra otros paquetes de r-universo que dependen de este paquete, agrupados por propietario/a. Esto te da una idea de lo activo que es el proyecto y de quién está participando.
```

File: content/blog/2023-02-24-runiverse-discovering-es/index.md [hunk 12]
```diff
@@ -0,0 +1,135 @@
...
+
+{{< figureforgallery src="../runiverse-discovering/package7.png" alt="Screenshot of recent development activity and users of a package." >}}
```
Comment (yabellini) [inline content/blog/2023-02-24-runiverse-discovering-es/index.md:124]:
```suggestion
{{< figureforgallery src="../runiverse-discovering/package7.png" alt="Ejemplo de actividad reciente de desarrollo y personas que usan el paquete." >}}
```

File: content/blog/2023-02-24-runiverse-discovering-es/index.md [hunk 13]
```diff
@@ -0,0 +1,135 @@
...
+
+## Conclusión
+
+Este post ha explicado diferentes formas en las que puedes utilizar [r-universo](https://r-universe.dev) para explorar los paquetes de R. La página [motor de búsqueda](https://r-universe.dev/search) te permite encontrar paquetes de cualquier parte del ecosistema que coincidan con determinadas palabras clave o criterios específicos. Es un buen lugar para informarse y hacerse una idea de lo que hay disponible en un dominio determinado.
```
Comment (yabellini) [inline content/blog/2023-02-24-runiverse-discovering-es/index.md:128]:
```suggestion
Este post ha explicado diferentes formas en las que puedes utilizar [r-universe](https://r-universe.dev) para explorar los paquetes de R. La página [motor de búsqueda](https://r-universe.dev/search) te permite encontrar paquetes de cualquier parte del ecosistema que coincidan con determinadas palabras clave o criterios específicos. Es un buen lugar para informarse y hacerse una idea de lo que hay disponible en un dominio determinado.
```

File: content/blog/2023-02-24-runiverse-discovering-es/index.md [hunk 14]
```diff
@@ -0,0 +1,135 @@
...
+
+Una vez que te familiarices con el ecosistema, puede que te interese hacer zoom en un conjunto específico de paquetes de un determinado grupo o investigador. La estructura de dominios de r-universo te permite explorar paquetes y artículos por organización, y descubrir trabajos relacionados con mantenedores, colaboradores o usuarios similares.
```
Comment (yabellini) [inline content/blog/2023-02-24-runiverse-discovering-es/index.md:130]:
```suggestion
Una vez que te familiarices con el ecosistema, puede que te interese hacer zoom en un conjunto específico de paquetes de un determinado grupo o investigador/a. La estructura de dominios de r-universe te permite explorar paquetes y artículos por organización, y descubrir trabajos relacionados con las personas que los mantienen, colaboran o los usan.
```

File: content/blog/2023-02-24-runiverse-discovering-es/index.md [hunk 15]
```diff
@@ -0,0 +1,135 @@
...
+
+Por último, para cada paquete R, r-universe proporciona una hermosa página de inicio que contiene abundante información en: `https://{owner}.r-universe.dev/{package}`. Esta página muestra exactamente de qué trata un paquete, quién lo está desarrollando, el estado del proyecto, y también proporciona toda la documentación y las instrucciones de instalación, muy bien representadas, para empezar a utilizar el paquete inmediatamente, sin complicaciones.
+
+[^1]: Más información sobre esta clasificación: [¿cómo se calcula la puntuación de la clasificación por paquetes?](https://github.com/r-universe-org/help#how-is-the-package-rank-score-calculated)
+[^2]: Entrada en el blog: [postdoc 1.0: manuales de paquetes HTML mínimos y despejados](/blog/2022/11/29/postdoc-docs/).
```
Comment (yabellini) [inline content/blog/2023-02-24-runiverse-discovering-es/index.md:135]:
```suggestion
[^2]: Entrada en el blog: [postdoc 1.0: manuales de paquetes HTML mínimos y claros](/blog/2022/11/29/postdoc-docs/).
```

**Review by yabellini — COMMENTED** [review]
I remove the gender mark and translated all the alt-text. 
I also go back to English the interface sections, because users will be found the name in English, but I add an explanation in Spanish.
I think the Spanish version is ready.

**Review by maelle — COMMENTED** [review]
I had forgotten to submit my review!

File: content/blog/2023-02-24-runiverse-discovering-es/index.md [hunk 16]
```diff
@@ -0,0 +1,135 @@
...
+### Toda la documentación
+
+Para que puedas empezar a utilizar los paquetes lo antes posible, r-universe ofrece toda la documentación disponible y la presenta de la forma más clara y concisa posible en la página de inicio.
+
+La mayoría de los paquetes de R tienen 3 formas de documentación: un archivo léeme, un manual de referencia (páginas de ayuda) y una serie de artículos (viñetas). Estos artículos suelen ser un buen punto de partida, y aparecen en la página principal del paquete, justo debajo de los descriptivos. Desde aquí puedes ver tanto el documento html/pdf renderizado, como el archivo fuente rmarkdown utilizado para (re)producir el artículo.
+
+{{< gallery >}}
+{{< figureforgallery src="../runiverse-discovering/package3.png" alt="Screenshot of r-universe package, listing included articles." >}}
+{{< figureforgallery src="../runiverse-discovering/package5.png" alt="Screenshot of r-universe package showing example readme content." >}}
+{{< figureforgallery src="../runiverse-discovering/package6.png" alt="Screenshot of r-universe package showing reference manual index." >}}
```
[the remark this answers is not in the document]
Comment (jeroen) [reply content/blog/2023-02-24-runiverse-discovering-es/index.md:109]: hmm I think I used the latest version

File: content/blog/2023-02-24-runiverse-discovering/index.md [hunk 17]
```diff
@@ -0,0 +1,137 @@
...
+
+Finally we arrive at the individual package level. Clicking on a package in search results or in a universe overview will lead you to homepage for that package. We can also link to this page directly via the url `https://{owner}.r-universe.dev/{package}` for example https://r-spatial.r-universe.dev/sf below.
+
+{{< figureforgallery src="package1.png" alt="Screenshot of r-universe package homepage" >}}
+
+These package pages are generated automatically and provide detailed information about each project, and all the information and resources to quickly install and try the package locally in R.
+
+### Package desciptives
+
+The initial section of the homepage shows descriptive data and links to resources from this package, such as source/binary files, the package manual and NEWS file. Further down we show installation instructions to install the package in R. Installation is quick and easy because we provide pre-compiled binaries for Windows and MacOS, so no special tools or setup is required on your end.
```
[the remark this answers is not in the document]
Comment (jeroen) [reply content/blog/2023-02-24-runiverse-discovering/index.md]: This entire post is aimed at the package shopper, not the package author.

**Pull request merged**